features/apps/src/main/feature/feature.xml

```diff
@@ -38,11 +38,9 @@
 
     <feature name="ddf-boot-features" version="${project.version}"
              description="Features that will be started on container startup.">
-        <!--Added prerequisite to kernel since it starts up features that must be started before others such as wrap and blueprint -->
-        <feature prerequisite="true">kernel</feature>
-        <feature prerequisite="true">platform-paxweb-jettyconfig</feature>
-        <!--Added prerequisite to make sure the security filter injector is running before any other contexts get registered-->
-        <feature prerequisite="true">security-services-app</feature>
+        <feature>kernel</feature>
```

Comment:
❓ We don't need to enforce blocking on `kernel` any more?

Reply:
Nope, only the wrap feature within the kernel feature.